src/test_helpers/utils.js

```diff
@@ -24,3 +25,13 @@ export function expectNoThrow (func) {
     expect.fail('no throw', 'throw', 'The function is expected to not throw.')
   }
 }
+
+/**
+ * @param promise - promise to check
+ * @param [err] - can be provided to check rejection for specific error type. Leave blank
+ * to ensure that promise was not rejected
+ * @return {*}
+ */
+export function shouldNotBeRejected (promise, err) {
+  return promise.should.not.be.rejectedWith(err)
```

Comment:
`expect` and `should` of the sinon should not be mixed up, please stick to `expect`